packages/e2fsprogs.rb

```diff
@@ -1,28 +1,32 @@
+# frozen_string_literal: true
+
 require 'package'
 
 class E2fsprogs < Package
   description 'e2fsprogs are ext2/3/4 file system utilities.'
   homepage 'http://e2fsprogs.sourceforge.net/'
-  version '1.45.6'
+  @_ver = '1.45.7'
+  version @_ver
   compatibility 'all'
-  source_url 'https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v1.45.6/e2fsprogs-1.45.6.tar.xz'
-  source_sha256 'ffa7ae6954395abdc50d0f8605d8be84736465afc53b8938ef473fcf7ff44256'
+  source_url "https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v#{@_ver}/e2fsprogs-#{@_ver}.tar.xz"
+  source_sha256 '62d49c86d9d4becf305093edd65464484dc9ea41c6ff9ae4f536e4a341b171a2'
 
-  binary_url ({
-    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.6-chromeos-armv7l.tar.xz',
-     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.6-chromeos-armv7l.tar.xz',
-       i686: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.6-chromeos-i686.tar.xz',
-     x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.6-chromeos-x86_64.tar.xz',
-  })
-  binary_sha256 ({
-    aarch64: '8c2bedd0fb8038b7e79bede2b7042754a6704fcec570433ef77bc9fb6c1b7745',
-     armv7l: '8c2bedd0fb8038b7e79bede2b7042754a6704fcec570433ef77bc9fb6c1b7745',
-       i686: '78505c168409d8239266de5d24d123b2199453f17255fc1d42b9d59e01539a99',
-     x86_64: 'e81555a711b57bed9e3a5a9850f207006ef4c5398d049f07793d314127616965',
-  })
+  binary_url({
+               aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.7-chromeos-armv7l.tar.xz',
+               armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.7-chromeos-armv7l.tar.xz',
+               i686: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.7-chromeos-i686.tar.xz',
+               x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/e2fsprogs-1.45.7-chromeos-x86_64.tar.xz'
+             })
+  binary_sha256({
+                  aarch64: '4deb49e04d94878c4a9e4977a5aa01937c379cf332b56b4e93524f6a891bdbc1',
+                  armv7l: '4deb49e04d94878c4a9e4977a5aa01937c379cf332b56b4e93524f6a891bdbc1',
+                  i686: 'a950f3395704adfc93ec3207cfd5d62ae1c1def8c6e70cfb83e0a113092034c8',
+                  x86_64: '37e6c64a587dd9d984194a0d22829a36c93d4fac413675b00bd0d77091e210c6'
+                })
```

Comment:
How did we start getting tabs all of a sudden?

Reply:
When I select them in the browser, those show up as spaces.

Reply:
That is a lot of extra spaces.  Can we clean that up?

Reply:
That indenting is as per rubocop. See slack mention.

Reply:
More or less this: : [Correctable] Layout/FirstHashElementIndentation: Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.

Reply:
new rubocop settings...